hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/CryptoUpdateSuite.java

```diff
@@ -132,6 +163,122 @@ final Stream<DynamicTest> idVariantsTreatedAsExpected() {
                         .newStakedAccountId("0.0.21")));
     }
 
+    private static final String[] ACCOUNTS_TO_HAVE_KEYS_ROTATED = {
+        "longZero", "autoCreated", "hollowAccount", "explicitAlias"
+    };
+    private static final UnaryOperator<String> ROTATION_TXN = account -> account + "KeyRotation";
```

Comment:
In a future refactor: This would make a fine HAPISpec utility: The four different ways of creating an account w/ EC key, assembled together and put in the registry.  I bet it could be used in multiple other tests to easily provide completeness.

Reply:
Ah yes, makes sense. 💯